portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,28 +27,52 @@
 public class DataServlet extends HttpServlet {
     
   private List<String> comments;
+  private List<String> names;
 
   @Override
   public void init() {
     comments = new ArrayList<String>();
+    names = new ArrayList<String>();
   }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("application/json");
 
+    List<String> namesAndComments = new ArrayList<String>();
+
+    assert names.size() == comments.size() : "error";
+    for(int index=0; index<names.size(); index++) {
```

Comment:
better to leave a space before and after the operators. int index = 0, index < names.size();